presentation/tests.py

```diff
@@ -1,3 +1,44 @@
+from django.core.urlresolvers import reverse
+from django.test import Client
 from django.test import TestCase
 
-# Create your tests here.
+from presentation.models import Presentation
+from warp.users.models import User
+
+
+class PresentationListTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.test_user = self.create_test_user()
+        self.presentation = Presentation(subject="subject",
+                                         author=self.test_user,
+                                         markdown="#Abcdefghijklmno",
+                                         html="<h1>Abcdefghijklmno</h1>")
+        for _ in range(0, 20):
+            self.presentation.save()
+
+    def test_get_presentation_list_page(self):
+        response = self.client.get(reverse('presentation:list'))
+
+        self.assertContains(response, self.presentation.subject)
```

Comment:
Or use the context value of response to assert equality with `self.presentation.subject` exactly.